LanguageFeatures/Inline-classes/syntax_A04_t01.dart

```diff
@@ -2,42 +2,46 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion A rule for <viewDeclaration> is added to the grammar, along with
-/// some rules for elements used in view declarations:
+/// @assertion A rule for <inlineClassDeclaration> is added to the grammar,
+/// along with some rules for elements used in inline class declarations:
 ///
-/// <viewDeclaration> ::=
-///   'view' 'class' <typeIdentifier> <typeParameters>?
-///       <viewPrimaryConstructor>?
-///       <interfaces>?
+/// <inlineClassDeclaration> ::=
+///   'inline' 'class' <typeIdentifier> <typeParameters>? <interfaces>?
 ///   '{'
-///     (<metadata> <viewMemberDeclaration>)*
+///     (<metadata> <inlineMemberDeclaration>)*
 ///   '}'
 ///
-/// <viewPrimaryConstructor> ::=
-///   '(' <type> <identifier> ')'
-///
-/// <viewMemberDeclaration> ::=
-///   <classMemberDefinition>
+/// <inlineMemberDeclaration> ::= <classMemberDefinition>
 /// ...
-/// The name of the representation in a view declaration that includes a
-/// <viewPrimaryConstructor> is the identifier id specified in there, and the
-/// type of the representation is the declared type of id.
+/// The name of the representation in an inline class declaration is the name id
+/// of the unique final instance variable that it declares, and the type of the
+/// representation is the declared type of id.
 ///
 /// @description Checks that static type of id is its declared type and there is
 /// a getter named id
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=inline-class
 
-view class View1(num id0) {
+inline class IC1 {
+  final num id0;
+  IC1(this.id0);
+}
+
+inline class IC2 {
+  final int _x;
+  IC2.n(int x) : this._x = x;
 }
 
-view class View2(int _x) {
+inline class IC3 {
+  final int _id = 0;
 }
 
 main() {
-  View1 v1 = View1(3.14);
-  v1.id0.isFinite;
-  View2 v2 = View2(42);
-  v2._x.isOdd;
+  IC1 ic1 = IC1(3.14);
+  ic1.id0.isFinite;
```

Comment:
It's safer to use `expectStaticType<Exactly<int>>` (the type of `id0` could be `Never`).

Reply:
Same consideration below.